### DISCUSSION TITLE
Filament community on Telegram

### DISCUSSION THREAD

**Discussion by HamidulloRahmonberdiyev** [filamentphp/filament#12545] [General]
Join members of the filament community on Telegram

Link: https://t.me/filament_community

**Comment by leandrocfe** [comment]
![image](https://github.com/filamentphp/filament/assets/3833889/d9027af4-fb47-48bc-a922-59db34ad2002)

https://discord.com/channels/883083792112300104/1234473480813281291/1234491574034104321
